extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,104 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
 }
 
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0);
+  width: 2.8em;
+  height: 2.8em;
+  top: 50%;
+  left: var(--percentage);
+  text-align: center;
+  line-height: 2.8em;
+  z-index: 2 !important;
+  font-weight: 500 !important;
+  color: white !important;
+  transition: transform 0.3s ease-in-out !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble.show {
+  transform: translate(-50%, -170%) scale(1) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 2.8em !important;
+  width: 2.8em !important;
```

Comment:
These values can be `100%` if we want them to take up the same width and height of the parent.